models/base/scratch/base_scratch.yml

```diff
@@ -1,8 +1,33 @@
 version: 2
 
 models:
-  - name: snowplow_media_player_interactions_this_run
-    description: '{{ doc("table_interactions_this_run") }}'
+  - name: snowplow_media_player_base_new_event_limits
+    description: '{{ doc("table_base_new_event_limits") }}'
+    columns:
+      - name: lower_limit
+        description: The lower `collector_tstamp` limit for the run
+      - name: upper_limit
+        description: The upper `collector_tstamp` limit for the run
+
+  - name: snowplow_media_player_base_sessions_this_run
+    description: '{{ doc("table_base_sessions_this_run") }}'
+    columns:
+      - name: session_id
+        description: '{{ doc("col_domain_sessionid") }}'
+        tags:
+          - primary-key
+        tests:
+          - unique
+          - not_null
+      - name: domain_userid
+        description: '{{ doc("col_domain_userid") }}'
+      - name: start_tstamp
+        description: The `collector_tstamp` when the session began
+      - name: end_tstamp
+        description: The `collector_tstamp` when the session ended
+
+  - name: snowplow_media_player_base_events_this_run
+    description: '{{ doc("table_base_events_this_run") }}'
```

Comment:
I know they were already missing, but collector_tstamp and event_in_session_index are both misisng descriptions.